### PR TITLE
adds ghetto/improvised tools (port)

### PR DESCRIPTION
**Pull request by boodaliboo**
Original PR: https://github.com/HippieStation/HippieStation/pull/11127
"Added improvised crowbars, multitool, wirecutter, welding tool, screwdriver and wrench craftable from the crafting menu."
all the tools have downsides
#### Changelog

:cl:  
rscadd: adds new improvised tools
imageadd: adds new improvised tools

/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
do0nt powergame

**Comment by Partheo** [comment]
You changed the RPD sprite as well as a bunch of other things. There's also a rogue pixel.
image

**Comment by boodaliboo** [comment]
god damnit

**Comment by boodaliboo** [comment]
> You changed the RPD sprite as well as a bunch of other things. There's also a rogue pixel.
> image

ill fix that

**Comment by boodaliboo** [comment]
> > You changed the RPD sprite as well as a bunch of other things. There's also a rogue pixel.
> > image
> 
> ill fix that

fixed

**Comment by boodaliboo** [comment]
i dont get the error on linters never seen a error there beofre

**Review by Partheo — APPROVED** [review]

File: code/game/objects/items/tools/crowbar.dm [hunk 1]
```diff
@@ -60,4 +60,19 @@
...
+	icon_state = "crowbar_ghetto"
+	force = 12 //same as large crowbar, but bulkier
+	w_class = WEIGHT_CLASS_BULKY
+
+/obj/item/crowbar/ghetto/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	if(prob(2))
+		to_chat(user, "<span class='danger'>[src] crumbles apart in your hands!</span>")
+		qdel(src)
+		return
+		...()
```
Comment (JamieD1) [inline code/game/objects/items/tools/crowbar.dm]:
```suggestion
		..()
```

**Comment by boodaliboo** [comment]
ill take a more indepth look at fixing this tommorow when i get home scream at me if i forgot because i most likely will

**Comment by alexkar598** [comment]
Not a fan of the multitool

**Comment by alexkar598** [comment]
or the rest of the tools if im being fucking honest

**Pull request closed without merging**